torch/_decomp/__init__.py

```diff
@@ -370,7 +370,6 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.norm,
             aten.ones,
             aten.ones_like,
-            aten.pixel_shuffle,
```

Comment:
Can you add this to the inductor decompositions to fix the inductor tests, or is the internal test matching `aten.pixel_shuffle` in an inductor pass?

Reply:
Oh, yes I'll add to the inductor decomps